ttnn/cpp/ttnn/tensor/layout/tensor_layout.cpp

```diff
@@ -0,0 +1,195 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "tensor_layout.hpp"
+
+namespace tt::tt_metal {
+
+namespace {
+size_t round_up(size_t value, size_t multiple) {
```

Comment:
There is already another implementation of round_up, please reuse

Reply:
Fair, will find and move to some common. Thanks